client/src/templates/layouts/NavigationLayout.js

```diff
@@ -137,27 +138,64 @@ const NavigationLayout = (props) => {
     </>
   );
 
+
+
+const [ isMobile, setMediaQuery ] = useState(false);
+
+useEffect(() => {
+  const mediaQuery = window.matchMedia(mq.phone.wide.max);;
```

Comment:
Not sure if this is the best approach. I think we should apply media queries within styled-components instead? That way it's easier to update and target styles based on media query. What do you think?

Reply:
Alright.
The blue box disappearing on tab view has always been the case, its not because of the navbar. i think that's for issue #300. 
I also purposely made the hamburger menu to appear at 768px because thats where the phone wide view ends. If it starts at 450px i dont think the desktop navbar will fit the tablet view

Reply:
Ah you're right, the landing page issue is related to #300. 

The navbar looks good and I'll merge this PR!